pkg/kubectl/describe.go

```diff
@@ -500,33 +500,8 @@ func describeContainers(pod *api.Pod, out io.Writer) {
 			fmt.Fprintf(out, "      %s:\t%s\n", name, quantity.String())
 		}
 
-		switch {
-		case state.Running != nil:
-			fmt.Fprintf(out, "    State:\tRunning\n")
-			fmt.Fprintf(out, "      Started:\t%v\n", state.Running.StartedAt.Time.Format(time.RFC1123Z))
-		case state.Waiting != nil:
-			fmt.Fprintf(out, "    State:\tWaiting\n")
-			if state.Waiting.Reason != "" {
-				fmt.Fprintf(out, "      Reason:\t%s\n", state.Waiting.Reason)
-			}
-		case state.Terminated != nil:
-			fmt.Fprintf(out, "    State:\tTerminated\n")
-			if state.Terminated.Reason != "" {
-				fmt.Fprintf(out, "      Reason:\t%s\n", state.Terminated.Reason)
-			}
-			if state.Terminated.Message != "" {
-				fmt.Fprintf(out, "      Message:\t%s\n", state.Terminated.Message)
-			}
-			fmt.Fprintf(out, "      Exit Code:\t%d\n", state.Terminated.ExitCode)
-			if state.Terminated.Signal > 0 {
-				fmt.Fprintf(out, "      Signal:\t%d\n", state.Terminated.Signal)
-			}
-			fmt.Fprintf(out, "      Started:\t%s\n", state.Terminated.StartedAt.Time.Format(time.RFC1123Z))
-			fmt.Fprintf(out, "      Finished:\t%s\n", state.Terminated.FinishedAt.Time.Format(time.RFC1123Z))
-		default:
-			fmt.Fprintf(out, "    State:\tWaiting\n")
-		}
-
+		describeStatus("State", state, out)
+		describeStatus("Last Termination State", status.LastTerminationState, out)
```

Comment:
`describeStatus("Last Termination State"...)` should only be called if `status.LastTerminationState` exists.